magclassic/templates/emails/prefest_faq.html

```diff
@@ -0,0 +1,31 @@
+If you are receiving this, you are registered for <a href="http://classic.MAGFest.org">MAGClassic</a>, occurring September 11-13, 2015. This FAQ should answer most your questions about the event. You may reach us at contact@magfest.org for any questions not covered here.
+
+<b> Will passes be sold at the door?</b>
+Yes. Full three-day passes will be available for $60, and one day passes will be available for $35 for Friday, $35 for Saturday, and $25 for Sunday. Children 5 & under are free, and children 12 & under receive a discount. We do not anticipate selling out, so you should have no problem purchasing a pass at-event.
+
+<b>Where do I go when I arrive? How do I pick up my pass?</b>
+The first step is to go to the registration desk, located in Upper Foyer near the parking garage entrance to pick up your pass. The registration desk is open from 10am - midnight Friday and Saturday, and 10am-1pm Sunday. If you pre-registered, simply show photo id such as a driver's license, military id, or student id to the registration staffers. If you do not have photo id, print out the confirmation e-mail you received when you pre-registered.
+
+<b>Which registration line do I use?</b>
+- Attendees who purchased a standard badge should use the "Pre-Registered Badge Pickup" line.
+- Attendees who did a group registration, or Marketplace vendors, should use the "Groups & Dealers" line.
```

Comment:
The ampersands are showing up weirdly here, but I'm assuming that's just Github's highlighting.